gpt_researcher/retrievers/serpapi/serpapi.py

```
@@ -28,7 +28,7 @@ def get_api_key(self):
         """
         try:
             api_key = os.environ["SERPAPI_API_KEY"]
-        except:
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)